lib/next/test/unit/units/normalize/normalizeHeaders.test.js

```diff
@@ -5,10 +5,8 @@ const {
 
 describe('normalizeHeaders', () => {
   describe('when given nothing', () => {
-    const headers = normalizeHeaders(undefined);
-
-    it('coerces to empty object', () => {
-      assert.deepEqual(headers, {});
+    it('throws upon invalid headers value', () => {
+      assert.throw(() => normalizeHeaders(undefined));
```

Comment:
Coercion is no longer a part of normalization so we must ensure we throw if `headers` is invalid value.